### PR TITLE
Document correct NewLineChars default values

### LINKED ISSUE #8315: XmlWriterSettings.NewLineChars defaults to "\n" on Linux, contrary to docs
### Description

On Linux, the default value of XmlWriterSettings.NewLineChars is "\n", even though the [documentation](https://docs.microsoft.com/dotnet/api/system.xml.xmlwritersettings.newlinechars?view=net-6.0) says it is "\r\n".

### Reproduction Steps

PowerShell 7.2.5 using .NET 6.0.6 on Debian:

```PowerShell
[System.Xml.XmlWriterSettings]::new().NewLineChars.GetEnumerator() | % { [int]$_ }
```

### Expected behavior

Should output

```
13
10
```

### Actual behavior

Outputs only

```
10
```

### Regression?

I don't know

### Known Workarounds

Assign XmlWriterSettings.NewLineChars after construction.

### Configuration

.NET 6.0.6 on Debian 11.3 amd64.
The issue does not occur on Windows.

### Other information

<https://github.com/dotnet/runtime/blob/7cca709db2944a09b4db6ca7b20c457ff260fb5a/src/libraries/System.Private.Xml/src/System/Xml/Core/XmlWriterSettings.cs#L159-L160>

<https://github.com/dotnet/runtime/blob/7cca709db2944a09b4db6ca7b20c457ff260fb5a/src/libraries/System.Private.Xml/src/System/Xml/Core/XmlWriterSettings.cs#L723>

### PR DESCRIPTION
**Pull request by wenz**
## Summary

Document NewLineChars default values on different operating systems.

Fixes #8315

### CODE REVIEW

File: xml/System.Xml/XmlWriterSettings.xml [hunk 1]
```diff
@@ -871,7 +871,7 @@ using (StreamWriter output =
...
 
--   The <xref:System.Xml.NewLineHandling.Replace> setting tells the <xref:System.Xml.XmlWriter> to replace new line characters with `\r\n`, which is the new line format used by the Microsoft Windows operating system. This helps to ensure that the file can be correctly displayed by the Notepad or Microsoft Word applications. This setting also replaces new lines in attributes with character entities to preserve the characters. This is the default value.
+-   The <xref:System.Xml.NewLineHandling.Replace> setting tells the <xref:System.Xml.XmlWriter> to replace new line characters with the character(s) specificed in the <xref:System.Xml.XmlWriterSettings.NewLineChars%2A> property. This setting also replaces new lines in attributes with character entities to preserve the characters.
```
[the remark this answers is not in the document]
Comment (wenz) [reply xml/System.Xml/XmlWriterSettings.xml:874]: Good point. The way this was phrased I understood it as "\r\n is the default value for the new line characters". Happy to put it back in, but possibly I'll reword it to make it more obvious that it refers to Replace.

[the remark this answers is not in the document]
Comment (wenz) [reply xml/System.Xml/XmlWriterSettings.xml]: @KalleOlaviNiemitalo Updated the wording. Thanks for the comment!

**Review by krwq — APPROVED** [review]
LGTM although consider mentioning this is correlated with https://learn.microsoft.com/en-us/dotnet/api/system.environment.newline?view=net-7.0

**Review by krwq — APPROVED** [review]
